apps/src/code-studio/viewAsRedux.js

```diff
@@ -1,24 +1,28 @@
 /**
- * Reducer for tracking whether we're currently viewing the page as a student
- * or as a teacher
+ * Reducer for tracking whether we're currently viewing the page as a participant
+ * or as a instructor
  */
 
 import {makeEnum, reload} from '@cdo/apps/utils';
 import {queryParams, updateQueryParam} from '@cdo/apps/code-studio/utils';
 
-export const ViewType = makeEnum('Student', 'Teacher');
+export const ViewType = makeEnum('Participant', 'Instructor');
 
 // Action types
 export const SET_VIEW_TYPE = 'viewAs/SET_VIEW_TYPE';
 
-export default function reducer(state = ViewType.Student, action) {
+export default function reducer(state = ViewType.Participant, action) {
   if (action.type === SET_VIEW_TYPE) {
-    const viewType = action.viewType;
-    if (!ViewType[viewType]) {
+    let viewType = action.viewType;
+    if (viewType === 'Teacher') {
+      viewType = 'Instructor';
+    } else if (viewType === 'Student') {
+      viewType = 'Participant';
```

Comment:
good call to handle these cases. will this cause the url to get updated?

Reply:
It does now :)